File: backend/app/Models/Item/ItemCategory.php
<?php

namespace App\Models\Item;

use Eloquent;
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

/**
 * @mixin Eloquent
 * @mixin Builder
 */
class ItemCategory extends Model
{
  protected $fillable = [
    'name',
    'code',
  ];

  public function items(): HasMany
  {
    return $this->hasMany(Item::class);
  }
}

File: UserPanel/Models/ItemCategory.cs
using System.Text.Json.Serialization;
namespace UserPanel.Models
{
    public class ItemCategory
    {
        [JsonPropertyName("id")] public required int Id { get; set; }
        [JsonPropertyName("name")] public required string Name { get; set; }
        [JsonPropertyName("code")] public required string Code { get; set; }
    }
}
